go/pkg/yolotree/yolotree.go

```diff
@@ -0,0 +1,22 @@
+package yolotree
```

Comment:
This package is not very relevant; it adds more complexity

You should implement this in the yolopb package as a method of *yolopb.Batch:

```go
func (b *yolopb.Batch) DisplayTreeFormat() {
```